docs/building.md

```diff
@@ -75,20 +75,20 @@ make out/trivialDAO_storage.tz \
   guardian_address=KT1QbdJ7M7uAQZwLpvzerUyk7LYkJWDL7eDh \
   governance_token_address=KT1RdwP8XJPjFyGoUsXFQnQo1yNm6gUqVdp5 \
   governance_token_id=0n \
-  now_val=`date +"%s"` \
+  current_level=100n \
```

Comment:
Looks like this document needs an update.

Reply:
:facepalm: Updated.